src/ui/public/agg_types/buckets/date_histogram.js

```diff
@@ -41,7 +41,8 @@ export default function DateHistogramAggType(timefilter, config, Private) {
     makeLabel: function (agg) {
       const output = this.params.write(agg);
       const params = output.params;
-      return params.field + ' per ' + (output.metricScaleText || output.bucketInterval.description);
+      const field = params.field || agg.params.field.displayName || '';
```

Comment:
Most other aggs ([example](https://github.com/elastic/kibana/blob/da04e108ded8ae7044ded2bb84f212d9c9eaba32/src/ui/public/agg_types/buckets/histogram.js#L19-L19)) accesses displayName directly, so it seemed safe to me. I suppose there might something special about the date histogram though. I can change it to use lodash's get if you'd like.